src/routes/balances/balances.controller.spec.ts

```diff
@@ -352,9 +352,7 @@ describe('Balances Controller (Unit)', () => {
         });
 
         await request(app.getHttpServer())
-          .get(
-            `/v1/chains/${chain.chainId}/safes/${safeAddress}/balances/${currency}`,
-          )
+          .get(`/v1/chains/${chain.chainId}/safes/${safeAddress}/balances/EUR`)
```

Comment:
Consider using `faker.finance.currencyCode()`

Reply:
Changed in [b69541f](https://github.com/safe-global/safe-client-gateway/pull/768/commits/b69541f62eedaf3a1a759dab61d4a1ad90206006)